lib/modules/datasource/pypi/index.spec.ts

```diff
@@ -125,6 +130,64 @@ describe('modules/datasource/pypi/index', () => {
       });
     });
 
+    it('supports Google Auth', async () => {
+      httpMock
+        .scope('https://someregion-python.pkg.dev/some-project/some-repo/')
+        .get('/azure-cli-monitor/json')
+        .matchHeader(
+          'authorization',
+          'Basic b2F1dGgyYWNjZXNzdG9rZW46c29tZS10b2tlbg==',
+        )
+        .reply(200, Fixtures.get('azure-cli-monitor-updated.json'));
+      const config = {
+        registryUrls: [
+          'https://someregion-python.pkg.dev/some-project/some-repo',
+        ],
+      };
+      googleAuth.mockImplementationOnce(
+        jest.fn().mockImplementationOnce(() => ({
+          getAccessToken: jest.fn().mockResolvedValue('some-token'),
+        })),
+      );
+      const res = await getPkgReleases({
+        ...config,
+        datasource,
+        packageName: 'azure-cli-monitor',
+      });
+      expect(res?.releases.pop()).toMatchObject({
```

Comment:
```suggestion
      expect(res).toMatchObject({
```
compare
 full object